docs/pages/versions/unversioned/sdk/admob.md

```diff
@@ -80,6 +80,32 @@ await AdMobRewarded.requestAdAsync();
 await AdMobRewarded.showAdAsync();
 ```
 
+### Testing
+
+- Here is the full list of Test IDs
+  - [iOS Test IDs](https://developers.google.com/admob/ios/test-ads)
+  - [Android Test IDs](https://developers.google.com/admob/android/test-ads)
+- Ensure you **never** load a real production ad in an Android Emulator or iOS Simulator. Failure to do this can result in a ban from the AdMob program.
```

Comment:
Oh, didn't know that!

Reply:
ha me either, until pillar valley got put on review. I downloaded the APK and dropped it in the emulator to test if turtle builds added the android package as a URI scheme for production Android apps -- I didn't have my Android phone on me at the time.

Got one of these a few hours later:

<img width="687" alt="Screen Shot 2020-08-12 at 2 46 56 PM" src="https://user-images.githubusercontent.com/9664363/90071664-d38cea00-dcaa-11ea-92dc-1e4f65193226.png">

I pushed a fix, then about a week later ads started showing up again -- no follow up email though.